Makefile

```
@@ -0,0 +1,6 @@
+local-webpage :
+	cp -rf README.md docs
+	mkdocs build --clean
```

Yes, if we use the docker image from mkdocs-material, we could replace this for:
```suggestion
	docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:6.2.4
```

---

Yep, makes sense